### ISSUE TITLE
How to import a sublime/textmate theme

### ISSUE THREAD

**Issue by Fanus** [thlorenz/brace#95]
I'm able to import a theme directly into Ace. But when I try do the same with brace, it doesn't seem to pick up the theme and I get a `theme-<themename>.js` 404 error in my browser.

Is importing a custom theme supported? Can a get a list of instructions on how go about import and theme into brace?

Regards

**Comment by danieltodonnell** [comment]
Yes, I came here for the exact same reason.  I want to define my own theme and reference it.

Even have a "bare-bones" theme that I can extend with css in the application would be close-to-ideal.

Example of my implementation below:
`        <AceEditor
          readOnly
          showGutter={false}
          className="json-display"
          theme="<CUSTOM THEME HERE>"
          mode="javascript"
          name="data"
          width="100%"
          tabSize={2}
          showPrintMargin={false}
          maxLines={Infinity}
          {...props}
        />`
